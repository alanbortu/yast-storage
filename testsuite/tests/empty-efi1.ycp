// testedfiles: helper1b.yh
{
    include "setup-system.yh";

    setup_system("empty-efi");

    include "helper1a.yh";

    import "ProductFeatures";

    ProductFeatures::SetBooleanFeature("partitioning", "try_separate_home", false);
    ProductFeatures::SetBooleanFeature("partitioning", "proposal_lvm", false);
    ProductFeatures::SetStringFeature("partitioning", "vm_desired_size", "30 GB");
    ProductFeatures::SetStringFeature("partitioning", "root_base_size", "20 GB");

    include "helper1b.yh";
}
