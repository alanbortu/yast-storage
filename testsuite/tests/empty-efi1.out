Dump	Proposal:
Dump	Create boot volume /dev/sda1 (156.88 MiB) with vfat
Dump	Create swap volume /dev/sda2 (517.72 MiB)
Dump	Create root volume /dev/sda3 (297.43 GiB) with ext4
